test/active_directory/windows_ca.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Pls dump the cert using x509 util to ensure that this only has the public part in in